chia/consensus/blockchain.py

```diff
@@ -95,16 +93,12 @@ class Blockchain:
     # peak of the blockchain
     _peak_height: Optional[uint32]
     # All blocks in peak path are guaranteed to be included, can include orphan blocks
-    __block_records: dict[bytes32, BlockRecord]
+    _block_records: dict[bytes32, BlockRecord]
     # all hashes of blocks in block_record by height, used for garbage collection
-    __heights_in_cache: dict[uint32, set[bytes32]]
+    _heights_in_cache: dict[uint32, set[bytes32]]
     # maps block height (of the current heaviest chain) to block hash and sub
     # epoch summaries
-    __height_map: BlockHeightMap
```

Comment:
It could be that originally we wanted this `BlockHeightMap` to be a singleton. But I think it could just cause more problems than it solves.

Reply:
The `.__` double underscore makes it a class variable, which makes it a singleton, so I thought maybe that was the original intent. But yeah, I also considered it might just be a misunderstanding/copying the (unnecessary?) patterns that already existed in `Blockchain`.

Reply:
Kyle correctly points out it's not a singleton... it's just an instance variable that's name-mangled at every use in a way that's class specific. This makes it even less plausibly useful here; and was probably just introduced by mistaken since I don't think this class has a subclass. Maybe it did at one point, but I'd be surprised.